src/Raven.Server/Config/Categories/ReplicationConfiguration.cs

```diff
@@ -42,5 +42,10 @@ public class ReplicationConfiguration : ConfigurationCategory
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Replication.MaxSizeToSendInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size? MaxSizeToSend { get; set; }
+        
+        [Description("The number of duplicate attachments that should be tracked for de-duplication per outgoing replication connection.")]
+        [DefaultValue(16*1024)]
+        [ConfigurationEntry("Replication.AttachmentDeduplicationBufferSize", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
"BufferSize" suggest it's about the size of some buffer in MB so when an attachment is bigger then it won't be a subject of de-duplication. 

I don't have good name for that though. My proposal are: 
- Replication.MaxNumberOfDeduplicatedAttachmentsInBatch
- Replication.MaxNumberOfAttachmentsTrackedForDeduplication